app/models/book_suggestion.rb

```diff
@@ -0,0 +1,5 @@
+class BookSuggestion < ApplicationRecord
+  validates :author, :title, :link, :publisher, :year, presence: true
+
+  belongs_to :user
```

Comment:
What about the user? Should he be able to access to his books suggestions

Reply:
@facucompe done! Added has_many :book_suggestions to User model.